README.md

```diff
@@ -137,6 +138,19 @@ sayHello('Test')()
 Added in v0.6.0
 ````
 
+### Run locally
+
+You can also preview the generated docs locally. `docs-ts` emits a file structure that can be rendered by the [jekyll](https://jekyllrb.com/) static site generator. Thus, we need to have this tool installed on the local system.
+We provide a `Dockerfile` and a `docker-compose.yml` file, so you don't have to worry about the details.
+
+From your project, just run the following command.
+
+`docker-compose --file node_modules/docs-ts/docker-compose.yml --project-directory . up --build`
```

Comment:
I could not test this. But what should come close is:
Run `docker-compose --file /path/to/local/docs-ts/docker-compose.yml --project-directory . up --build` inside any local `fp-ts` project. Assuming you have checked out this branch in `/path/to/local/docs-ts`.

This worked for me.